client/my-sites/people/followers-list/followers.jsx

```diff
@@ -0,0 +1,270 @@
+			this.props.totalFollowers <= this.props.followers.length ||
+			MAX_FOLLOWERS <= this.props.followers.length
+		);
+	}
+
+	render() {
+		const key = deterministicStringify( omit( this.props.query, [ 'max', 'page' ] ) );
+		const listClass = classNames( {
+			'bulk-editing': this.state.bulkEditing,
+			'people-invites__invites-list': true,
+		} );
+
+		if ( this.noFollowerSearchResults() ) {
+			return (
+				<NoResults
+					image="/calypso/images/people/mystery-person.svg"
+					text={ this.props.translate( 'No results found for {{em}}%(searchTerm)s{{/em}}', {
+						args: { searchTerm: this.props.query.search },
+						components: { em: <em /> },
+					} ) }
+				/>
+			);
+		}
+
+		let emptyTitle;
+		if ( this.siteHasNoFollowers() ) {
+			if ( this.props.query && 'email' === this.props.query.type ) {
+				emptyTitle = preventWidows(
+					this.props.translate( 'No one is following you by email yet.' )
+				);
+			} else {
+				emptyTitle = preventWidows( this.props.translate( 'No WordPress.com followers yet.' ) );
+			}
+			return <EmptyContent title={ emptyTitle } action={ this.renderInviteFollowersAction() } />;
+		}
+
+		let headerText;
+		if ( this.props.totalFollowers ) {
+			headerText = this.props.translate(
+				'You have %(number)d follower',
+				'You have %(number)d followers',
+				{
+					args: { number: this.props.totalFollowers },
+					count: this.props.totalFollowers,
+				}
+			);
+
+			if ( this.props.type === 'email' ) {
+				headerText = this.props.translate(
+					'You have %(number)d follower receiving updates by email',
+					'You have %(number)d followers receiving updates by email',
+					{
+						args: { number: this.props.totalFollowers },
+						count: this.props.totalFollowers,
+					}
+				);
+			}
+		}
+
+		let followers;
+		if ( this.props.followers.length ) {
+			if ( this.props.query.search && this.props.totalFollowers ) {
+				headerText = this.props.translate(
+					'%(numberPeople)d Follower Matching {{em}}"%(searchTerm)s"{{/em}}',
+					'%(numberPeople)d Followers Matching {{em}}"%(searchTerm)s"{{/em}}',
+					{
+						count: this.props.followers.length,
+						args: {
+							numberPeople: this.props.totalFollowers,
+							searchTerm: this.props.query.search,
+						},
+						components: {
+							em: <em />,
+						},
+					}
+				);
+			}
+
+			const infiniteListConditionals = {
+				fetchingNextPage: this.props.fetching,
+				lastPage: this.isLastPage(),
+			};
+
+			followers = (
+				<InfiniteList
+					{ ...infiniteListConditionals }
+					key={ key }
+					items={ this.props.followers }
+					className="followers-list__infinite is-people"
+					ref={ this.infiniteList }
+					fetchNextPage={ this.fetchNextPage }
+					getItemRef={ this.getFollowerRef }
+					renderLoadingPlaceholders={ this.renderPlaceholders }
+					renderItem={ this.renderFollower }
+					guessedItemHeight={ 126 }
+				/>
+			);
+		} else {
+			followers = this.renderPlaceholders();
+		}
+
+		const downloadListLink =
```

Comment:
Nit: would be cool if we could use a more intelligent way to build a query string for this URL. We have `addQueryArgs()` for example.

Feel free to ignore; I know this was like this before this PR.